lock_api/build.rs

```diff
@@ -0,0 +1,16 @@
+fn main() {
```

Comment:
Adding a build script will make building a bit slower as the build script has to be compiled and the main crate has to wait on running the build script.

Reply:
Yeah. An alternative approach without such drawbacks is to use the options feature. If the maintainer prefers that approach, I'm happy to switch to using it.

Reply:
I personally don't mind this: the build script has no dependencies and can be compiled in parallel with other crates, so it is usually not noticable.